setup.py

```diff
@@ -18,7 +18,8 @@ def read(fname):
     license='MIT',
     python_requires='>=2.7.9,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
     install_requires=[
-        'mkdocs>=0.17'
+        'mkdocs>=0.17',
+        'ruamel.yaml',
```

Comment:
mkdocs makes use of PyYAML (id prefer to stick with the base requirements instead of adding bloat (i know ruamel is preferred these days tho)